content/hi/docs/reference/glossary/configmap.md

```diff
@@ -0,0 +1,18 @@
+---
+title: कॉन्फिगमैप
+id: configmap
+date: 2018-04-12
+full_link: /docs/concepts/configuration/configmap/
+short_description: >
+  गैर-गोपनीय डेटा को कुंजी-मूल्य जोड़े में संग्रहीत करने के लिए उपयोग की जाने वाली API ऑब्जेक्ट। एनवायरनमेंट वैरिएबल, कमांड-लाइन आर्ग्यूमेंट्स, या वॉल्यूम में कॉन्फ़िगरेशन फ़ाइलों के रूप में उपभोग किया जा सकता है।
```

Comment:
IMO it is better to transliterate `key-value pair` instead of translating as it is technical term that most may be familiar with and transliterating allows more to quickly understand/associate with the term. Thoughts, WDYT @Garima-Negi , @amolmote? I am also open to suggestions by other contributors as well. Thanks (It would be great if others could share their opinion before this suggestion is committed/discarded)
```suggestion
  गैर-गोपनीय डेटा को की-वैल्यू पेअर में संग्रहीत करने के लिए उपयोग की जाने वाली API ऑब्जेक्ट। एनवायरनमेंट वैरिएबल, कमांड-लाइन आर्ग्यूमेंट्स, या वॉल्यूम में कॉन्फ़िगरेशन फ़ाइलों के रूप में उपभोग किया जा सकता है।
```